salt/utils/gitfs.py

```diff
@@ -491,9 +491,7 @@ def _val_cb(x, y):
         self._cache_basename = "_"
         if self.id.startswith("__env__"):
             try:
-                self._cache_basename = self.get_checkout_target().replace(
-                    "/", "-"
-                )  # replace '/' with '-' to not cause trouble with file-system
+                self._cache_basename = self.get_checkout_target()
```

Comment:
What happens with instantiations that already have `-` caches? Will those caches just go stale and get invalidated? Meaning once they upgrade they'll just get a cache miss?

Reply:
When upgrading the cache is cleared (at least that is what I remember from comments or the PR's from Chad and Garth), so the old '-' cache entries will be gone